docs-src/concepts/what-is-the-temporal-cli.md

```diff
@@ -28,3 +26,187 @@ Workflow, show Workflow History, and Signal Workflow).
 - [temporal server](/cli/server)
 - [temporal task-queue](/cli/task-queue/)
 - [temporal workflow](/cli/workflow/)
+
+## Installation
+
+Temporal CLI can be installed through several different methods. While most of them can be used across all operating systems, please note that Homebrew is macOS-exclusive.
+
+For more information, see our guide to [running a Development Cluster](/clusters/how-to-install-temporal-cli).
+
+### cURL
+
+Run the following command to install the latest version of Temporal CLI.
+
+`curl -sSf https://temporal.download/cli.sh | sh`
+
+### Homebrew
+
+Run the following command in a new terminal window to install CLI for macOS.
+
+`brew install temporal`
+
+### GitHub releases
+
+Download and extract the latest release from [GitHub releases](https://github.com/temporalio/cli/releases).
+
+### CDN
+
+To install the Temporal CLI from CDN:
+
+1. Select the binary with the corresponding platform and architecture of your system.
+2. Download the binary.
+3. Extract the downloaded archive.
+4. Add the `temporal` binary to your system PATH.
+
+## Starting the Temporal Server
+
+Run the command provided below to start the Temporal Server.
+This will automatically start the Web UI.
+
+```bash
+temporal server start-dev
+```
+
+At this point you should have a server running on `localhost:7233` and a web interface at <http://localhost:8233>.
+A default Namespace has also been created.
+
+Run individual commands to interact with the local Temporal server.
+
+```bash
+temporal operator namespace list
+temporal workflow list
+```
+
+## Configuration
+
+Use the help flag to see a full list of CLI options:
+
+```bash
+temporal server start-dev --help
+```
+
+### Environmental variables
```

Comment:
Environment variables apply to the whole CLI - not just the `server start-dev` command.

Reorg the content appropriately - this is sitting right between two `server` command info bits.